virt/virt-additional-security-privileges-controller-and-launcher.adoc

```diff
@@ -1,27 +1,27 @@
 :_content-type: ASSEMBLY
 [id="virt-additional-security-privileges-controller-and-launcher"]
-= Additional security privileges granted for kubevirt-controller and virt-launcher
+= Security policies
 include::_attributes/common-attributes.adoc[]
 :context: virt-additional-security-privileges-controller-and-launcher
 
 toc::[]
 
-The `kubevirt-controller` and virt-launcher pods are granted some SELinux policies and Security Context Constraints privileges that are in addition to typical pod owners. These privileges enable virtual machines to use {VirtProductName} features.
```

Comment:
I would keep this part in some form " that are in addition to typical pod owners." as we did not tackle this yet.

Reply:
Thanks, @xpivarc - but I'm not 100% sure that I understand the original meaning of that phrase. Does this capture it? I'd add it to the end of the introduction (before the "About workload security" module).

`* SELinux policies and SCCs that apply to typical pod owners also apply to {VirtProductName} pods.`
or maybe
`* Security policies that apply to typical pod owners also apply to {VirtProductName}.`

Reply:
We have and grant custom SELinux policy to virt-launcher and the same goes with SCC. These 2 things are typically not available to a typical pod owner.

Reply:
Ah! Thank you for clarifying. I made an update that should address this. Please let me know if you have any outstanding concerns. @xpivarc